packages/runtime/src/runtimes/react/controls/style.ts

```diff
@@ -63,6 +65,14 @@ function useStyleControlCssObject(
           borderBottomRightRadius: borderRadiusToString(borderRadius?.borderBottomRightRadius) ?? 0,
           borderBottomLeftRadius: borderRadiusToString(borderRadius?.borderBottomLeftRadius) ?? 0,
         }),
+        ...(properties.includes(StyleControlProperty.TextStyle) && {
+          ...(textStyle?.fontFamily && { fontFamily: textStyle.fontFamily }),
+          ...(textStyle?.letterSpacing && { letterSpacing: textStyle.letterSpacing }),
+          ...(textStyle?.fontSize && { fontSize: fontSizeToString(textStyle.fontSize) ?? '16px' }),
+          ...(textStyle?.fontWeight && { fontWeight: textStyle.fontWeight }),
+          textTransform: textStyle?.textTransform ?? [],
```

Comment:
Don't CSS style values always have to be a string, `null`, or sometimes a number?

Reply:
`textTransform` and `fontStyle` could be an array:
<img width="489" alt="image" src="https://user-images.githubusercontent.com/13066728/181034430-17e277cd-c76a-4342-92c0-bd99d906f0d5.png">

Reply:
I didn't know that! That's cool